hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/TokenExpiryInfoV1SecurityModelSuite.java

```diff
@@ -98,6 +98,8 @@ public List<Stream<DynamicTest>> getSpecsInSuite() {
         return List.of(updateExpiryInfoForToken(), updateExpiryInfoForTokenAndReadLatestInfo());
     }
 
+    // Currently cannot be adapted to v2 because we have to explore an asynchronous process or prior approval.
+    // See HIP-755 and HIP-756 for more context
```

Comment:
When referencing HIPs in comments, I suggest also providing the name of the HIP.
```
// Currently cannot be adapted to v2 because we have to explore an asynchronous process or prior approval.
// See HIP-755 (Schedule Service System Contract) and HIP-756 (Contract Scheduled Token Create)
```

Reply:
Updated! Plus two more test fixes